book/modules/appendix1-exercises.tex

```diff
@@ -17,18 +17,50 @@
 
 			\item $\frac{x}{y}=1$
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
```

Comment:
Let's change these to be shorter. `Linear equation` and `Not a linear equation because of the ... term`